### ISSUE TITLE
text issue

### ISSUE THREAD

**Issue by Owenm05** [Areso/1255-burgomaster#693]
![image](https://user-images.githubusercontent.com/43863985/195638073-a515d0ed-bfd2-4727-8dc1-22ee9d8fbde2.png)
does not make sense

**Comment by Owenm05** [comment]
“Smm facility”

**Comment by Areso** [comment]
No, it means faculty - a group of university departments concerned with a major division of knowledge
SMM faculty - faculty of SMM. Social Media Management.

**Comment by Areso** [comment]
the picture is bad, though

**Comment by Owenm05** [comment]
The wizard confused me
